### PR TITLE
Expose `color.get_xyz_coords` in public API

### PR DESCRIPTION
**Pull request by lagru**
## Description

For now [it's mentioned once in the docstring of `skimage.color.lab2rgb`](https://scikit-image.org/docs/dev/search.html?q=get_xyz_coords&check_keywords=yes&area=default) but it's probably intended to be part of our public API. So it should be part of our public API documentation too!

See also #6688 which directly references this function.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

**Review by mkcor — APPROVED** [review]
Indeed! Thanks, @lagru.

**Comment by stefanv** [comment]
I'm not going to merge this for now; there are no other similar functions, and the name is not descriptive enough IMO, so would like to discuss first.

**Comment by mkcor** [comment]
> I'm not going to merge this for now; there are no other similar functions, 

Right, I noticed this afterwards when posting https://github.com/scikit-image/scikit-image/issues/1185#issuecomment-1399319643.

> and the name is not descriptive enough IMO, so would like to discuss first.

Ok, this makes total sense!

File: skimage/color/colorconv.py
Comment (lagru) [inline skimage/color/colorconv.py:1]: Slightly off-topic: I'm also curious about all these matrix definitions in `colorconv.py` that are imported into `skimage.color` but not documented at all. Some of them seem to be never used (e.g. `gray_from_rgb`).

https://github.com/scikit-image/scikit-image/blob/c90bfccbc930262eebd1149b10660bfa4f56ec61/skimage/color/colorconv.py#L411-L419

**Comment by lagru** [comment]
>  there are no other similar functions, and the name is not descriptive enough IMO, so would like to discuss first.

Agreed, the name could be better. What do you think about `xyz_tristimulus_values` (see 4c1be589db8bd024a46debf1add72ac9c007de23).

**Comment by lagru** [comment]
But I'm perfectly happy, to go the other route and hide / deprecate this function. Less maintenance for us... :tipping_hand_person:

**Comment by mkcor** [comment]
> But I'm perfectly happy, to go the other route and hide / deprecate this function. Less maintenance for us... tipping_hand_person

I'm :+1: on hiding this function. Thanks!

**Comment by stefanv** [comment]
I think we should still merge the other changes here, even if we decide not to make this function public.

Whether or not to make it public, I just don't know how widely this is *in image processing*? Perhaps it fits better in a color science package, such as

https://colour.readthedocs.io/en/v0.3.11/tutorial.html#convert-to-tristimulus-values

But, I'm not familiar enough with the functionality to make the call.

**Comment by jni** [comment]
@lagru We're discussing this at community meeting... Technically `get_xyz_coords` was public, and a [code search confirms that it is used in the wild](https://github.com/search?q=get_xyz_coords&type=code), so we need to either deprecate it, keep the alias, or just not change the name at all...

**Comment by lagru** [comment]
It's not a function that I expect to be a big maintenance burden. It's mainly a wrapper to document and expose the content of a dict. Seeing that we are unsure, my suggestion is to err on the side of caution: Refactor to `xyz_tristimulus_values` with a deprecation cycle and make it public in `skimage.color`.

**Comment by stefanv** [comment]
That works, thanks @lagru. I like the new name.

**Comment by lagru** [comment]
This is ready for review / merge in my opinion.

File: skimage/color/colorconv.py [hunk 1]
```diff
@@ -500,6 +506,75 @@ def hsv2rgb(hsv, *, channel_axis=-1):
...
+    formula
+
+    .. math:: X = x / y
+
+    .. math:: Y = 1
+
+    .. math:: Z = (1 - x - y) / y
+
+    The only exception is the illuminant "D65" with aperture angle 2° for
+    backward-compatibility reasons.
```
Comment (lagru) [inline skimage/color/colorconv.py:555]: I couldn't figure out where the original values for this case come from. There is 

https://github.com/scikit-image/scikit-image/blob/8495b071ff81f3f1c5c4d8a44cb0387a81092066/skimage/color/colorconv.py#L492

which points to 

https://github.com/scikit-image/scikit-image/blob/8495b071ff81f3f1c5c4d8a44cb0387a81092066/skimage/color/colorconv.py#L446-L448

I'm unsure where these values are derived from. A git blame investigation wasn't really fruitful yet. So for this PR I'd just ignore it...

**Review by stefanv — APPROVED** [review]

**Comment by stefanv** [comment]
Thanks, Lars!

**Pull request merged**